stackrox/stackrox
# chore(ui): Delete ROX_NETPOL_FIELDS feature flag in ui

## Description

Deleted from backend in #6530

Noticed browser console warnings during manual testing of something else.

> EnvVar ROX_NETPOL_FIELDS not found in the backend list, possibly stale?

## Checklist

## Testing Performed

1. `yarn lint`
2. `yarn build`
3. `yarn start` in ui to see warning before delete and see no warning after delete:
    * Visit /main/violations: click **90-Day Image Age** and then click **Policy**
    * Visit /main/policy-management/policies: click **90-Day Image Age**
